apps/common/lookups.py

```diff
@@ -167,6 +164,25 @@ def prepare_lookup_df(self) -> pd.DataFrame:
             df = df.rename(columns={self.id_fields[0]: "lookup_value"})
         return df
 
+    def prepare_column_for_lookup(self, column):
+        # Always use a string lookup, because the lookup may be against columns of mixed data types
+        column = column.astype(str)
+
+        # Replace non-breaking spaces and tabs with space
+        column = column.str.replace("\xa0", " ")
```

Comment:
Thanks @chrispreee , do we maybe need to include other similar unrequited spaces ? Searching gave me the following similar options :
```
\xa0: No-break space
\u200b: Zero-width space
\u200c: Zero-width non-joiner
\u200d: Zero-width joiner
\u202f: Narrow no-break space
\u2002: En space
\u2003: Em space
\u2009: Thin space
\u2007: Figure space
\u205f: Medium mathematical space
\u3000: Ideographic space
\u00ad: Soft hyphen
```

Reply:
Good catch thanks Girum! Done as discussed.